### PR TITLE
Add option to output interactive learning stories in e2e format

### LINKED ISSUE #3410: optionally output interactive learning stories in e2e format
In interactive learning, we already have user input and the intent, validated by the user. We could output these stories in end-to-end format if the user desires so that they don't have to write e2e stories or turn regular ones into e2e ones on their own.

Perhaps this could happen under the flag `rasa interactive --e2e`.

### PR DESCRIPTION
**Pull request by chidauri**
**Proposed changes**:
- Fixes #3410 

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by chidauri** [comment]
Today i learnt, Dictionaries are global :100:

**Comment by msamogh** [comment]
Thanks for the PR, we'll give it a review as soon as possible.

File: rasa/core/training/interactive.py [hunk 1]
```diff
@@ -78,6 +78,8 @@
     "domain": "domain.yml",
 }
 
+SAVE_IN_E2E = {"value": False}
```
Comment (tmbo) [inline rasa/core/training/interactive.py]: is there any reason this needs to be a dict and can't just be a boolean?

File: rasa/cli/interactive.py [hunk 2]
```diff
@@ -29,6 +29,9 @@ def add_subparser(
         "Rasa model by chatting.",
     )
     interactive_parser.set_defaults(func=interactive)
+    interactive_parser.add_argument(
+        "--e2e", action="store_true", help="save file in e2e format"
```
Comment (tmbo) [inline rasa/cli/interactive.py]:
```suggestion
        "--e2e", action="store_true", help="Save story files in e2e format. In this format user messages will be included in the stories."
```

File: rasa/core/training/interactive.py (hunk 1 above)
Comment (chidauri) [reply rasa/core/training/interactive.py]: Actually i need a global variable, and dict works fine but boolean doesn't in my case :(

Comment (tmbo) [reply rasa/core/training/interactive.py]: do you know why it doesn't work? (or can you tell me more about what you need it for?)

Comment (chidauri) [reply rasa/core/training/interactive.py]: i am using server_args to pass e2e, and updating it here inside `run_interactive_learning` function, if i do something like `SAVE_IN_E2E = True` it doesn't affect the local variable, so i need a dictionary
https://github.com/chidauri/rasa/blob/d8b2fe1fdd857dc1b2b233c30b3d3ad99a25faca/rasa/core/training/interactive.py#L1556

Comment (tmbo) [reply rasa/core/training/interactive.py]: ah right, you need to declare it to be global:

at the beginning of the `run_interactive_learning` function, just put `global SAVE_IN_E2E `

**Pull request merged**